internal/migrations/clusterlogforwarder/migrate_inputs.go

```diff
@@ -6,6 +6,21 @@ import (
 )
 
 func MigrateInputs(namespace, name string, spec loggingv1.ClusterLogForwarderSpec, logStore *loggingv1.LogStoreSpec, extras map[string]bool, logstoreSecretName, saTokenSecret string) (loggingv1.ClusterLogForwarderSpec, map[string]bool, []loggingv1.Condition) {
+	for i, input := range spec.Inputs {
+		if input.Receiver != nil {
+			if input.Receiver.HTTP != nil && len(input.Receiver.Type) == 0 {
```

Comment:
strings are initialized so you can just compare this to `""`

Reply:
done